D/D Single File.sublime-build

```diff
@@ -0,0 +1,13 @@
+{
+	"shell_cmd": "dmd $file",
```

Comment:
Fair enough, my only concern was some people use the Universal D binaries which you can "activate" in your bashrc. Specifically I did this in testing, and since the binaries aren't in the normal PATH it would only work with shell_cmd

Reply:
In that case we just need $file quoted, I believe

Reply:
`cmd` is only problematic on Windows because it doesn't find `dmd.exe` and the like on the PATH. You'd need a shell_cmd for that or a Windows-specific override for `cmd`.

Reply:
The "activate" thing is probably related to environment variables and most likely either to be activated by the login shell (in which case ST should pick it up regardless) or, if it's per-directory, either have ST already be launched from that directory by the shell or not something to be run by a session-global application like ST.

However, considering you're talking about bashrc, this is much more likely the be the former.